oaknational/Oak-Web-Application
# LESQ-1144: reorder home page tabs

## Description

Music year: 1978

- Reorders home page
- Teachers links to teachers home page
- Updates urls file

## Issue(s)

Fixes #LESQ-1144

### Acceptance Criteria
- Teachers, curric, AI, pupils
- Leave new on AI
- “Teachers” in the top right takes you to /teachers


## How to test

1. Go to https://deploy-preview-2938--oak-web-application.netlify.thenational.academy
2. Home page should be the teachers tab, the order of the tabs should be teachers, curriculum, ai, pupils
3. Navigate away from home, then click teachers in app header, should navigate to /teachers

## Screenshots


https://github.com/user-attachments/assets/a5f61e56-29bd-4fe1-9bb8-c36f3699e06d



## Checklist
